packages/shoreline/src/components/toast/toast-stack.tsx

```diff
@@ -53,6 +53,7 @@ export function ToastStack(props: ToastStackProps) {
               id={t.id}
               loading={t.type === 'loading'}
               variant={(t as any).variant as any}
+              ariaProps={toastOptions?.ariaProps || t.ariaProps}
```

Comment:
Did you try this approach? I think it has the same behavior since we pass the `toastOptions` to the `useToaster` hook
```suggestion
              {...t.ariaProps}
```

Reply:
got it, in this case, I would recommend something like that, what do you think?
```suggestion
              {...t.ariaProps}
              {...toastOptions?.ariaProps}
```

Reply:
Sounds good ma friend! 🚀

Reply:
To make that change, we must set Toast props to `role` and `aria-live` and remove the property `ariaProps` itself.

Reply:
That's intentional? Just to clarify the suggestion

Reply:
it's not necessary to specify those properties on the interface because we are spreading the props to a `div` element it should work as expected. What you can do to be more typesafe is to extend the ToastProps from `React.ComponentPropsWithoutRef<'div'>`

Reply:
I don't know if I understand this right, but I made a new commit.

[Check this out!](https://github.com/vtex/shoreline/pull/1856/commits/2f7da58aaba6508e7e4487a61736124450fc9ea4)

Reply:
What I mean is that the `ToastOptions['ariaProps']` are all native props such as `role` and `aria-live` so is not necessary to create a `ariaProps: ToastOptions['ariaProps']` to pass those props to the Toast component, you can just spread them as you did in this new solution